### PR TITLE
port can_msgs to ament build system

### PR DESCRIPTION
**Pull request by mathias-luedtke**
extracted from #359 

I am not sure, if we should add `AMENT_IGNORE` files and remove them later.

(@JWhitleyAStuff )

### CODE REVIEW

**Comment by JWhitleyWork** [comment]
Did you intend to target this PR to the `melodic-devel` branch?

I would suggest using `COLCON_IGNORE` files.

File: can_msgs/package.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">.
+<package format="3">.
```
Comment (JWhitleyWork) [inline can_msgs/package.xml]: After the `<?xml version="1.0"?>`, this should have `<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>`. `ament_xmllint` can then use the XSD to validate the contents of this file.

**Comment by mathias-luedtke** [comment]
> Did you intend to target this PR to the melodic-devel branch?

No, I will retarget it :)

**Review by JWhitleyWork — APPROVED** [review]
LGTM.

**Comment by mathias-luedtke** [comment]
released: https://github.com/ros/rosdistro/pull/22878

**Comment by JWhitleyWork** [comment]
Awesome! Thanks. Now on to everything else!

**Pull request merged**